.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Contributing
+
+If you discover issues, have ideas for improvements or new features,
+please report them to the [issue tracker][1] of the repository or
+submit a pull request. Please, try to follow these guidelines when you
+do so.
+
+## Issue reporting
+
+* Check that the issue has not already been reported.
+* Check that the issue has not already been fixed in the latest code
+  (a.k.a. `master`).
+* Be clear, concise and precise in your description of the problem.
+* Open an issue with a descriptive title and a summary in grammatically correct,
+  complete sentences.
+* Report the versions of `rubocop-rspec`, as well as the output of `rubocop -V`
+* Include any relevant code to the issue summary.
+
+## Pull requests
+1. Fork the project.
+2. Create a feature branch.
+3. Commit your changes.
```

Comment:
I’d say that “commit your changes” should come *after* “add tests” and “add changelog entry”.